__tests__/app.test.js

```diff
@@ -182,6 +183,75 @@ describe("CORE: GET - /api/articles/:article_id", () => {
             expect(dates).toBeSorted({ descending: true })
           });
       });
+      test("200: Topic query within endpoint returns articles filtered by specific topic", () => {
+        return request(app)
+          .get("/api/articles?topic=mitch")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles.length).not.toBe(0)
+            body.articles.forEach((article) => {
+                expect(article).toHaveProperty("topic", ('mitch'))
+            })        
+          });
+      });
+      test("200: Invalid topic query within endpoint returns empty array", () => {
+        return request(app)
+          .get("/api/articles?topic=faketopic")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body).toEqual({articles: []})
+          });
+      });
+      test("200: Sort_by query within endpoint returns articles sorted by specific column in default order", () => {
+        return request(app)
+          .get("/api/articles?sort_by=article_id")
+          .expect(200)
+          .then(({ body }) => {
+          
+            expect(body.articles.length).not.toBe(0)
+            const ids = body.articles.map((id) => {
+              return id.article_id
+            })        
+            expect(ids).toBeSorted({ descending: true })
+          })
+        })
+        test("200: Invalid sort by query within endpoint returns default sort (DESC BY DATE)", () => {
```

Comment:
Again here your description mentions that this query is invalid so would you expect a status 200 🤔